core/src/main/groovy/noe/tomcat/configure/SecureHttpConnectorTomcat.groovy

```diff
@@ -20,6 +20,7 @@ public class SecureHttpConnectorTomcat extends ConnectorTomcatAbstract<SecureHtt
   //
   // SSL common
   private boolean sslEnabled
+  private String protocol
```

Comment:
Protocol is present in parent already. This is no point to have if defined again in child.